### ISSUE TITLE
max_parallel_workers > 0 prevents upgrade from 1.4.1/2 to 1.5.0/1 on FreeBSD 12.1 and PostgreSQL 11.6

### ISSUE THREAD

**Issue by overhacked** [timescale/timescaledb#1615]
**Workaround**
Temporarily set `max_parallel_workers = 0` in `postgresql.conf` before running `ALTER EXTENSION timescaledb UPDATE;`. Restore to original value afterwards.

**Relevant system information:**
 - OS: FreeBSD 12.1-RELEASE-p1
 - PostgreSQL version (output of `postgres --version`): 11.6
 - TimescaleDB version (output of `\dx` in `psql`): 1.4.1
 - Installation method: `pkg`

**Describe the bug**
After upgrading the TimescaleDB shared libraries to version 1.5.1, the `ALTER EXTENSION timescaledb UPDATE;` command returns the following error:
```
ERROR:  extension "timescaledb" version mismatch: shared library version 1.5.1; SQL version 1.4.1
```

**To Reproduce**
1. Upgrade TimescaleDB: `pkg upgrade -y timescaledb`
2. Attempt `ALTER EXTENSION …`, which returns error `loader version out-of-date`
3. Restart PostgreSQL: `service postgresql restart`
4. Attempt `ALTER EXTENSION …`, which returns error `extension "timescaledb" version mismatch`

*See detailed console session, below, under **Screenshots***

**Expected behavior**
Successful upgrade of database.

**Actual behavior**
Version mismatch error requiring downgrade of TimescaleDB to continue operations.

**Screenshots**
```console
root@postgresql:~/timescaledb-1.5.1-pkg # pkg upgrade -Uy timescaledb
Checking integrity... done (0 conflicting)
The following 1 package(s) will be affected (of 0 checked):

Installed packages to be UPGRADED:
        timescaledb: 1.4.1 -> 1.5.1

Number of packages to be upgraded: 1

The process will require 1 MiB more space.

[postgresql] [1/1] Upgrading timescaledb from 1.4.1 to 1.5.1...
[postgresql] [1/1] Extracting timescaledb-1.5.1: 100%

root@postgresql:~ # sudo -u postgres psql -X -h /var/run/postgresql -d zabbix
psql (11.6)
Type "help" for help.

zabbix=# ALTER EXTENSION timescaledb UPDATE;
ERROR:  loader version out-of-date
HINT:  Please restart the database to upgrade the loader version.
zabbix=# \q
root@postgresql:~ # service postgresql restart
2020-01-07 15:06:24 UTC [58829] LOG:  listening on IPv6 address "::", port 5432
2020-01-07 15:06:24 UTC [58829] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2020-01-07 15:06:24 UTC [58829] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2020-01-07 15:06:24 UTC [58829] LOG:  redirecting log output to logging collector process
2020-01-07 15:06:24 UTC [58829] HINT:  Future log output will appear in directory "/var/log/postgresql".
root@postgresql:~ # sudo -u postgres psql -X -h /var/run/postgresql -d zabbix
psql (11.6)
Type "help" for help.

zabbix=# ALTER EXTENSION timescaledb UPDATE;
ERROR:  extension "timescaledb" version mismatch: shared library version 1.5.1; SQL version 1.4.1
CONTEXT:  parallel worker
```

**Comment by overhacked** [comment]
Confirmed this is a 1.5.0+ regression. I was able to successfully `UPDATE` from 1.4.1 to 1.4.2 in a test environment.

**Comment by overhacked** [comment]
Added **Workaround** section above based on successful Slack support with @cevian.

**Comment by svenklemm** [comment]
Fixed by #3288. Feel free to reopen if you still experience the issue after upgrading to a version that has the fix.
